source/hardware/controllers/numark_party_mix.rst

```diff
@@ -0,0 +1,114 @@
+Numark Party Mix
+================
+
+`Manufacturer's product page <https://www.numark.com/product/party-mix>`_ · `Manufacturer's user manual <https://www.numark.com/images/product_downloads/Party_Mix_-_User_Guide_-_v1.1.pdf>`_ · `Forum thread <https://mixxx.discourse.group/t/numark-party-mix-midi-mapping/16712>`_
+
+.. versionadded:: 2.3.2
+
+Overview
+--------
+
+.. figure:: ../../_static/controllers/numark_party_mix.png
```

Comment:
where did you get this picture from? We can't rip pictures straight from the hardwares manual because the manufacturer has the copyright over the picture.

Reply:
made one myself now